### PR TITLE
Add new php71 ruleset.

### PR DESCRIPTION
**Pull request by dereuromark**
Ruleset for https://github.com/spryker/code-sniffer/pull/118

**ON HOLD** as this has to be applied manually per major or new module for now!

### CODE REVIEW

File: Spryker/ruleset.xml [hunk 1]
```diff
@@ -28,6 +28,14 @@
     <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHintSpacing"/>
     <rule ref="SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue"/>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
+        <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversableParameterTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversablePropertyTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversableReturnTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.UselessParameterAnnotation"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.UselessDocComment"/>
```
Comment (PhilinTv) [inline Spryker/ruleset.xml]: We continue to comment code, even with typehints.

Comment (dereuromark) [reply Spryker/ruleset.xml]: Yes. For now.

Comment (PhilinTv) [reply Spryker/ruleset.xml]: Do you have any other plans?

Comment (dereuromark) [reply Spryker/ruleset.xml]: what do u have in mind?

**Comment by pushokwhite** [comment]
Has been fully merged to https://github.com/spryker/code-sniffer/pull/147

**Pull request closed without merging**